### ISSUE TITLE
Forgot Appeal Key

### ISSUE THREAD

**Issue by Liefernando1** [UTRS2/utrs#583]
How to recover Wikipedia Unblock Ticket Request System? I had forget appeal key.

**Comment by weeklyd3** [comment]
I'm not a developer of this project, but I think only administrators or those with the appeal key can view the appeal. Someone might query the database here, though, but I'm not sure.

Next time, I recommend storing the appeal key somewhere safely such as in a file on your computer.

**Comment by Liefernando1** [comment]
Then, what does this mean?

![Screenshot_2022-10-06-14-30-30-853_com android chrome](https://user-images.githubusercontent.com/104975946/194230700-52b31e82-5417-4ee1-8979-30c83a69c062.jpg)

**Comment by dqwiki** [comment]
Closing in favor of #46
